### DISCUSSION TITLE
Battle royale mode

### DISCUSSION THREAD

**Discussion by spacekotek** [ppy/osu#29356] [Ideas]
So recently i hosted a small battle royale event on lazer, we had a map pool and after each map one person was kicked out of the lobby.
It was pretty fun and i think this kind of thing would be nice to have as a kind of gamemode that takes advantage of lazer's lobby sizes.
I saw a similar post but i dont like that idea.

The way i see it:
- There is a lobby on multi that starts every few hours, everyone can join, but it is locked once it starts.

- The lobby has a queue of some amount of maps (we tested 10, it lasted for about an hour with 30 second breaks)
Where to take maps from? I think the ones that were in daily challenge can be used with the same difficulty scaling but selected randomly from all weeks.

- After each map, a certain amount of players who did the worst are moved to spectators, maybe with a result screen showing their score/placement/anything else.

- After the final map the winning player gets a +stat on profile. Maybe can also give a +stat to a certain top % of players?

**Comment by Skylandia** [comment]
Sounds like the kind of thing someone would make a bot for
